### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by MohammedAkramU7** [ahmedkaludi/easy-table-of-contents#803]
Ref ticket:- https://magazine3.in/conversation/306341?folder_id=21

PHP Fatal error: Uncaught Error: Call to undefined method ezTOC::InlineCountingCSS() in /var/www/vhosts/httpdocs/wp-content/plugins/easy-table-of-contents/includes/class-eztoc-widgetsticky.php:433\nStack trace:\n#0 /var/www/vhosts/httpdocs/wp-includes/class-wp-widget.php(394): ezTOC_WidgetSticky->widget()\n#1 /var/www/vhosts/httpdocs/wp-includes/widgets.php(845): WP_Widget->display_callback()\n#2 /var/www/vhosts/httpdocs/wp-content/themes/neve/views/sidebar.php(25): dynamic_sidebar()\n#3 /var/www/vhosts/httpdocs/wp-content/themes/neve/inc/views/base_view.php(52): include('...')\n#4 /var/www/vhosts/httpdocs/wp-content/themes/neve/inc/views/layouts/layout_sidebar.php(78): Neve\\Views\\Base_View->get_view()\n#5 /var/www/vhosts/httpdocs/wp-includes/class-wp-hook.php(324): Neve\\Views\\Layouts\\Layout_Sidebar->sidebar()\n#6 /var/www/vhosts/httpdocs/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()\n#7 /var/www/vhosts/ht...

**Comment by askcode21** [comment]
Same concern: https://wordpress.org/support/topic/wrong-function-name-2/

**Comment by Sdaud1950** [comment]
Same concern: https://magazine3.in/conversation/307518?folder_id=29

**Comment by Smith-Cruise** [comment]
same +1

**Comment by pauljacobson** [comment]
Here is a more complete version of the error for a site on WordPress.com:

```
PHP Fatal error:  Uncaught Error: Call to undefined method ezTOC::InlineCountingCSS() in /srv/htdocs/wp-content/plugins/easy-table-of-contents/includes/class-eztoc-widgetsticky.php:433
Stack trace:
#0 /wordpress/core/6.6.1/wp-includes/class-wp-widget.php(394): ezTOC_WidgetSticky->widget(Array, Array)
#1 /wordpress/core/6.6.1/wp-includes/widgets.php(845): WP_Widget->display_callback(Array, Array)
#2 /wordpress/themes/pub/ixion/sidebar.php(15): dynamic_sidebar('sidebar-1')
#3 /wordpress/core/6.6.1/wp-includes/template.php(810): require_once('/wordpress/them...')
#4 /wordpress/core/6.6.1/wp-includes/template.php(745): load_template('/wordpress/them...', true, Array)
#5 /wordpress/core/6.6.1/wp-includes/general-template.php(136): locate_template(Array, true, true, Array)
#6 /wordpress/themes/pub/ixion/page.php(50): get_sidebar()
#7 /wordpress/core/6.6.1/wp-includes/template-loader.php(106): include('/wordpress/them...')
#8 /wordpress/core/6.6.1/wp-blog-header.php(19): require_once('/wordpress/core...')
#9 /wordpress/core/6.6.1/index.php(17): require('/wordpress/core...')
#10 {main}
  thrown in /srv/htdocs/wp-content/plugins/easy-table-of-contents/includes/class-eztoc-widgetsticky.php on line 433
```

**Comment by Sdaud1950** [comment]
https://magazine3.in/conversation/308024?folder_id=29

**Comment by Sanjeevsetu** [comment]
This issue has been fixed and pushed in latest updated version 2.0.69.1. Please try upgrading and know us.
